indico/modules/events/editing/client/js/components/EditableSubmissionButton.jsx

```diff
@@ -18,14 +18,18 @@ import {Translate} from 'indico/react/i18n';
 import {FinalFileManager} from './FileManager';
 import {fileTypePropTypes} from './FileManager/util';
 
+import {getFileTypes} from '../selectors';
+
 export default function EditableSubmissionButton({
   eventId,
   contributionId,
+  contributionCode,
   uploadURL,
   fileTypes,
   type,
   submitRevisionURL,
 }) {
+  fileTypes = getFileTypes({staticData: {fileTypes, contributionCode}});
```

Comment:
missing `const`

Reply:
not missing, it's reassigning a value coming from the arguments

Reply:
Ahh.. I was searching for it :P For some reason I didn't notice it

Reply:
isn't it just overwriting because of line 25/28?